.github/workflows/dec-tests.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .[test]
+        pip install platformdirs
```

Comment:
Remember to move this into `setup.cfg` is this is an official new library we need.

Reply:
Ok i seem to be failing to add this dependecy, how does the setup.cfg file function work? I basically just added it to install_requires under libbs. The CI is still always not having platformdirs installed

Reply:
@Flipout50, my guy, you edited an example directory setup.cfg. You need to update the one in the root.

Reply:
Omg 🤦‍♂️

Reply:
Yo, this line is still here